### ISSUE TITLE
Fusioncatcher build fails

### ISSUE THREAD

**Issue by apeltzer** [nf-core/rnafusion#478]
### Description of the bug

![image](https://github.com/nf-core/rnafusion/assets/2359510/f33bce82-a328-4937-b222-4c18c124850c)

### Command used and terminal output

```console
nextflow run nf-core/rnafusion -r master -profile bi,XXX (this is a singularity profile), without home mounting.
```

**Comment by apeltzer** [comment]
@fmalmeida FYI

**Comment by rannick** [comment]
I suspect it has to do with the home mounting. Fusioncatcher does a lot of explicit path coding. Can you test with a standard singularity mount?

**Comment by apeltzer** [comment]
Yes, I can give it a try (y)

**Comment by apeltzer** [comment]
Hm still doesnt work I fear. 
![image](https://github.com/nf-core/rnafusion/assets/2359510/0fa8ce7e-d4a5-4cea-8e7b-f1dc4297bd5d)

a.) STARfusion build error:
nhmmscan --noali --cut_ga --dfamtblout /tmp/BRphdvSPd7 --cpu=24 homo_sapiens_dfam.hmm ref_annot.cdsplus.fa
Error, cmd: dfamscan.pl -fastafile ref_annot.cdsplus.fa -hmmfile homo_sapiens_dfam.hmm -dfam_outfile __dfam_ref_annot.cdsplus.fa/dfam.out --masking_thresh --cpu 24 died with ret 6400 No such file or directory at /usr/local/src/STAR-Fusion/ctat-genome-lib-builder/util/../lib/Pipeliner.pm line 186.
        Pipeliner::run(Pipeliner=HASH(0x5589ff7c0788)) called at /usr/local/src/STAR-Fusion/ctat-genome-lib-builder/util/dfam_repeat_masker.pl line 84
b.) Fusion Report
![image](https://github.com/nf-core/rnafusion/assets/2359510/e21cebd9-a70e-4b01-b3bc-fa3ce4c8753b)

c.) Fusion Catcher Build: (also fails with 127 exit code, not fond -- I have to check thats the problem there still). 

cc @fmalmeida FYI

**Comment by rannick** [comment]
Trying to reproduce, but with dev. I manage to run fusion-report, I would assume your error there is due to a access problem. Are you providing the correct cosmic username and password (quiagen ones)?

**Comment by apeltzer** [comment]
> Trying to reproduce, but with dev. I manage to run fusion-report, I would assume your error there is due to a access problem. Are you providing the correct cosmic username and password (quiagen ones)?

Yes, and I already checked that they are accurate (and they are)... :-(

**Comment by rannick** [comment]
Can I close this? Do you have any news @apeltzer ?

**Comment by apeltzer** [comment]
No let's see if the coming updates fix it

**Comment by nvnieuwk** [comment]
Has this issue been solved?

**Comment by rannick** [comment]
Downloading from s3 should solve the issue, closing for now
